ibis/expr/types/generic.py

```diff
@@ -2223,9 +2223,23 @@ class NullColumn(Column, NullValue):
 
 
 @public
-def null():
-    """Create a NULL/NA scalar."""
-    return ops.NULL.to_expr()
+def null(type: dt.DataType | str | None = None) -> Value:
+    """Create a NULL/NA scalar.
```

Comment:
```suggestion
    """Create a NULL scalar.
```

Let's keep terminology consistent where possible.